pkg/subctl/cmd/createbroker.go

```diff
@@ -80,5 +84,31 @@ var createBrokerCmd = &cobra.Command{
 			panic(err.Error())
 		}
 		fmt.Printf("There are %d pods in the cluster\n", len(pods.Items))
+
+		// Create CRDs
+		fmt.Printf("Creating CRDs\n")
+		apiextensionsClient, err := apiextensionsclientset.NewForConfig(config)
+		//_, err = apiextensionsClient.ApiextensionsV1beta1().CustomResourceDefinitions().Create(createClustersCRD())
+		created_clusters_crd, err := apiextensionsClient.ApiextensionsV1beta1().CustomResourceDefinitions().Create(createClustersCRD())
+		fmt.Printf("Created CRD:\n", created_clusters_crd)
+
+		// Wait for CRD creation
+		err = wait.Poll(5*time.Second, 60*time.Second, func() (bool, error) {
```

Comment:
This is missing from #14, so it would be useful to resurrect in another PR. I’m not sure how *necessary* it is in `subctl create broker` itself, since we don’t use the CRDs there, but perhaps we should have a separate `subctl` command to wait for broker readiness (as a general concept — the user shouldn’t know what’s involved).